server/service/hosts.go

```diff
@@ -889,6 +889,51 @@ func (svc *Service) ListHostDeviceMapping(ctx context.Context, id uint) ([]*flee
 	return svc.ds.ListHostDeviceMapping(ctx, id)
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// MDM
+////////////////////////////////////////////////////////////////////////////////
+
+type getHostMDMRequest struct {
+	ID uint `url:"id"`
+}
+
+type getHostMDMResponse struct {
+	Err error `json:"error,omitempty"`
+	*fleet.HostMDM
+}
+
+func getHostMDM(ctx context.Context, request interface{}, svc fleet.Service) (interface{}, error) {
+	req := request.(*getHostMDMRequest)
+	mdm, err := svc.MDMData(ctx, req.ID)
+	if err != nil {
+		return getHostMDMResponse{Err: err}, nil
+	}
+	return getHostMDMResponse{HostMDM: mdm}, nil
+}
+
+type getHostMDMSummaryResponse struct {
+	fleet.AggregatedMDMData
+	Err error `json:"error,omitempty"`
+}
+
+type getHostMDMSummaryRequest struct {
+	TeamID   *uint  `query:"team_id,optional"`
+	Platform string `query:"platform,optional"`
+}
+
+func getHostMDMSummary(ctx context.Context, request interface{}, svc fleet.Service) (interface{}, error) {
+	var (
+		req  = request.(*getHostMDMSummaryRequest)
+		resp = getHostMDMSummaryResponse{}
+		err  error
+	)
+	resp.AggregatedMDMData, err = svc.AggregatedMDMData(ctx, req.TeamID, req.Platform)
```

Comment:
Tiny nit, but for consistency (with most endpoints, I believe, and with getHostMDM above), I'd recommend:

```suggestion
	mdmData, err := svc.AggregatedMDMData(ctx, req.TeamID, req.Platform)
```

And avoid the `var` block above (by using `:=` for the `req`), and use `return getHostMDMSummaryResponse{AggregatedMDMData: mdmData}, nil` at the end, much like the response struct is created inline when returning an error.